### ISSUE TITLE
`trust find`, like `proof find` but for trust declarations

### ISSUE THREAD

**Issue by Minoru** [crev-dev/cargo-crev#352]
I'm running cargo-crev 0.17, installed via `cargo install`, on Debian Bullseye.

A user I trust on a "medium" level disappeared from `cargo crev id query trusted`. (I initially attributed that to me changing that user's level from "low" to "medium", but I think it was really caused by `cargo crev fetch all` that I ran about the same time.) I had to play with `--depth` to understand that this is caused by someone's negative review, and then had to run `grep -R '    id: ID' ~/.cache.crev` to find out whose review it was.

The issue would've been easier to resolve if Crev had a command that listed all users' trust declarations for a particular ID. I think it should be some called `cargo crev trust find`, but I haven't fully groked how CLI is organized, so perhaps this functionality should be added to `cargo crev proof find` instead.

**Comment by dpc** [comment]
I agree a command like this would be really useful. And as usually, determining the CLI is the most difficult part. Could be also `cargo crev id describe <id>` or something.
